packages/playwright/src/worker/testInfo.ts

```diff
@@ -292,6 +296,9 @@ export class TestInfoImpl implements TestInfo {
           }
         }
 
+        for (const attachment of attachments ?? [])
```

Comment:
I understand that this is where you move attachments collected for step into testinfo, but this code does not work for user steps, so let's not fork the behavior.

Reply:
reverted